static/app/views/issueDetails/groupReplays/useReplaysForRegressionIssue.tsx

```diff
@@ -0,0 +1,99 @@
+import {useCallback, useEffect, useMemo, useRef, useState} from 'react';
+import * as Sentry from '@sentry/react';
+import {Location} from 'history';
+
+import {getSampleEventQuery} from 'sentry/components/events/eventStatisticalDetector/eventComparison/eventDisplay';
+import {ALL_ACCESS_PROJECTS} from 'sentry/constants/pageFilters';
+import {Event, type Group, type Organization} from 'sentry/types';
+import EventView from 'sentry/utils/discover/eventView';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {DEFAULT_SORT} from 'sentry/utils/replays/fetchReplayList';
+import useApi from 'sentry/utils/useApi';
+import useCleanQueryParamsOnRouteLeave from 'sentry/utils/useCleanQueryParamsOnRouteLeave';
+import {REPLAY_LIST_FIELDS} from 'sentry/views/replays/types';
+
+function useReplaysForRegressionIssue({
+  group,
+  location,
+  organization,
+  event,
+}: {
+  event: Event;
+  group: Group;
+  location: Location;
+  organization: Organization;
+}) {
+  const now = useRef(new Date(Date.now()).toISOString());
```

Comment:
don't need to pass anything to get now
```suggestion
  const now = useRef(new Date().toISOString());
```